lib/components/profile/badge/Badges.tsx

```
@@ -4,66 +4,59 @@ import {
+
+  const { openBottomSheet } = useBottomSheet();
+
   // Map badgeData and mark unlocked badges
   const badgesToShow: (BadgeDetails & { isUnlocked: boolean })[] = badgeData.map(
     (badge) => {
       const unlocked = badges?.find((b) => b.name === badge.name && b.check);
       return {
         ...badge,
         isUnlocked: !!unlocked,
-      };
+      }; View
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)